feature/experimental/security/aaa/kne_tests/tls_authentication_over_grpc_test/tls_authentication_over_grpc_test.go

```diff
@@ -134,6 +134,26 @@ func TestAuthentication(t *testing.T) {
 			if err != nil {
 				t.Fatal(err)
 			}
+			
```

Comment:
gofmt is complaining about the whitespace on this line

Reply:
> gofmt is complaining about the whitespace on this line

Removed the whitespace